packages/popmotion-pose/src/inc/accessibility.ts

```diff
@@ -0,0 +1,17 @@
+let prefersReducedMotion = false;
+
+if (typeof window !== 'undefined') {
+  const motionMediaQuery = window.matchMedia('(prefers-reduced-motion)');
+
+  const setReducedMotionPreferences = () => {
+    prefersReducedMotion =
+      motionMediaQuery.media === 'not all'
+        ? false
+        : motionMediaQuery.matches ? true : false;
```

Comment:
`motionMediaQuery.matches` is already a boolean so this is literally casting a boolean to boolean, can be shortened to
```js
prefersReducedMotion = motionMediaQuery.media === 'not all'
  ? false
  : motionMediaQuery.matches
```

Reply:
also - cant find the justification for special handling `"not all"`, i might be missing something though, in all my tests when this condition is met `motionMediaQuery.media === 'not all'` then this is also true `motionMediaQuery.matches === false`, suggesting u could use just a simple
```js
prefersReducedMotion = motionMediaQuery.matches
```

Reply:
Yeah excellent points

Reply:
@InventingWithMonster just a side note, it might be nice to configure the prefersReducedMotion as an option since not all browsers implement it, we could instead make it a user preference in a UI somewhere.